.gitlab-ci.yml

```diff
@@ -30,6 +30,27 @@ test-features:
   script:
     - cargo test --all --all-features --all-targets --locked
 
+integration-tests:
```

Comment:
I think we should aim to change the workdir to be directly `chopsticks`, and do everything from inside there, including building the runtime, which could be another yarn script.

Reply:
Since this could potentially also be its own repo, so we should treat is as completely isolated from the rest of the repo.

Reply:
I am not too much in favor to move the directory into the workdir. First, I do not like, if the root directory is super full. Second, the “project” aims to provide some tests. I like the strict separation of logic and testing.

Reply:
That is not what I am suggesting. I was suggesting setting the `WORKDIR` to be `integration-tests/chopsticks` instead of `cd`ing into it. But I could not find a keyword that GitLab would support, so I guess `cd`ing into it is fine. I was not suggesting to copy or move anything 😄

Reply:
Ah okay sorry. Yeah I could also not find anything. That is why I `cd` into the directory.